### PR TITLE
...Now runs translated

### PR DESCRIPTION
**Pull request by gabi-250**

### CODE REVIEW

File: interp.py [hunk 1]
```diff
@@ -2,13 +2,29 @@
...
+    return main, None
+
+def jitpolicy(driver):
+    from rpython.jit.codewriter.policy import JitPolicy
+    return JitPolicy()
+
+jit_driver = jit.JitDriver (
+    greens = [],
+    reds = []
+    #get_printable_location = get_location
```
Comment (vext01) [inline interp.py]: Commented line. Remove?

File: llvm_wrapper.py [hunk 2]
```diff
@@ -6,17 +6,26 @@
 
 class CConfig(object):
     _includes = "llvm-c/Core.h llvm-c/BitReader.h"
-    output, error = Popen("llvm-config --libs core native --system-libs", 
-                    shell=True, stdout=PIPE, stderr=PIPE).communicate()
+    libs = Popen("llvm-config --libs core native --system-libs",
```
Comment (vext01) [inline llvm_wrapper.py:9]: This is nitpicking, but usually when you break a line inside parenthesis, you line up the next line with the open parenthesis. E.g.:

```
foo(this_is_a_long_line, with_many_arguments, lalala, lalala,
    next_line_aligned_like_this)
```

**Comment by vext01** [comment]
Other than the small thing I pointed out, looks good.

Remember to write a description, even if it is only short :)

**Comment by vext01** [comment]
Looks good. Would you like to squash any commits?

**Comment by gabi-250** [comment]
Should we squash any?

**Pull request merged**